modules/hal_nordic/nrfx/CMakeLists.txt

```diff
@@ -58,6 +59,7 @@ zephyr_library_sources_ifdef(CONFIG_SOC_NRF5340_CPUNET ${MDK_DIR}/system_nrf5340
 zephyr_library_sources_ifdef(CONFIG_SOC_NRF9160        ${MDK_DIR}/system_nrf9160.c)
 
```

Comment:
I think that for now, until the corresponding memory region is excluded from use in Zephyr and `NRF_FICR_NS` becomes actually used, `NRF_SKIP_FICR_NS_COPY_TO_RAM` should be defined in this file for `CONFIG_SOC_NRF5340_CPUAPP` and `CONFIG_SOC_NRF9160`.

Reply:
is this related to nrfx2.6.0 or can be made in separate PR? I'm not familiar with this topic.

Reply:
It's a new feature that comes with the MDK included in nrfx 2.6.0 (see [here](https://github.com/zephyrproject-rtos/hal_nordic/pull/106/files#diff-5fec5f44d9e363f8a7befe0695b5a91b1f8c9ee3c2d968df350f14de8feb8931R121-R123)), so I think it should be addressed in this PR.

Reply:
done